niceman/interface/tests/test_retrace.py

```diff
@@ -24,4 +24,12 @@ def test_retrace(reprozip_spec2):
                 '--spec', reprozip_spec2,
                 ]
         main(args)
-        assert_in("reading spec file " + reprozip_spec2, log.lines)
\ No newline at end of file
+        assert_in("reading spec file " + reprozip_spec2, log.lines)
+
+
+def test_retrace_to_output_file(reprozip_spec2):
+    with make_tempfile() as outfile:
+        args = ['retrace',
+                '--spec', reprozip_spec2,
+                '--output-file', outfile]
+        main(args)
```

Comment:
please check then that outfile was created, and ideally even loaded.  Otherwise it just tests that it doesn't crash

Reply:
> Otherwise it just tests that it doesn't crash

Right, it just tests that the crash in #146 isn't triggered.

> and ideally even loaded

By "loaded", do you just mean content was written to the file?

Reply:
see how to load a specification and access its basic properties e.g. here https://github.com/ReproNim/niceman/blob/master/niceman/interface/install.py#L81